blueprint-integration/src/main/resources/META-INF/spring/extender/extender-context.xml

```diff
@@ -1,12 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <beans xmlns="http://www.springframework.org/schema/beans" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xsi:schemaLocation="http://www.springframework.org/schema/beans
-http://www.springframework.org/schema/beans/spring-beans.xsd">
-
+						classpath:org/springframework/beans/factory/xml/spring-beans.xsd">
```

Comment:
Doesn't the mapping above avoid having to change the XML files?

Reply:
Indeed, but then I need to add the Spring xsd resources in two places:
- `blueprint-integration`
- `alfresco-dynamic-extensions-repo`

And so I did